index.html

```diff
@@ -245,5 +245,47 @@ <h4>Skills</h4>
             </li>
         </ul>
     </section>
+    <!--Starting form section-->
+    <section class="about">
+        <div class="contact-form">
+            <h3 class="form">I'm always interested in hearing about new projects, so if you would like to chat please</h3>
+            <div class="con-form">
+                <form action="#" method="post" class="request" >
+                    <input type="text" placeholder="Full name" maxlength="30" required>
+                    <input type="email" placeholder="Email address" maxlength="30" required>
+                    <textarea name="request" id="request" placeholder="Enter text area" maxlength="500" required></textarea>
+                    <div class="see mbili">
+                        <button type="submit">Get in touch</button>
```

Comment:
- [ ] kindly configure your form spree properly if i send  my data am getting an error message see screenshot 
`what am getting`
![Screenshot from 2022-07-06 03-51-01](https://user-images.githubusercontent.com/66230306/177581531-27e6c1e3-05fc-4613-83ee-37a48e02c3e9.png)

`what im suppose to get`
![Screenshot from 2022-07-06 03-52-08](https://user-images.githubusercontent.com/66230306/177581627-af4077d6-a4b4-4dd9-a32c-796172740ce4.png)
see this [article](https://formspree.io/html/) for further understandings